app/src/lib/feature-flag.ts

```diff
@@ -105,3 +105,4 @@ export const enableCommitDetailsHeaderExpansion = () => true
 export const enableDiffCheckMarksAndLinkUnderlines = enableBetaFeatures
 
 export const enableDiffCheckMarks = enableDiffCheckMarksAndLinkUnderlines
+export const enableGroupDiffCheckmarks = enableDevelopmentFeatures
```

Comment:
As noted in the PR body, this is not true feature flagging since the non-group diff checkmark is now different. But, will make it so that unnecessarily wide controls without checkmark controls doesn't ship to beta.